colossalai/inference/modeling/policy/nopadding_baichuan.py

```diff
@@ -100,3 +101,10 @@ def module_policy(self):
     def postprocess(self):
         init_to_get_rotary(self.model.model)
         return self.model
+
+    def to_rpc_param() -> str:
+        return "NoPaddingBaichuanModelInferPolicy"
```

Comment:
I guess you might like `__class__.__name__`, so that prevent hard-coding

Reply:
Thx!